qutebrowser/misc/editor.py

```diff
@@ -124,6 +124,6 @@ def edit(self, text):
         self._proc.error.connect(self.on_proc_error)
         editor = config.get('general', 'editor')
         executable = editor[0]
-        args = [self._filename if arg == '{}' else arg for arg in editor[1:]]
+        args = [arg.replace('{}', self._filename) if '{}' in arg else arg for arg in editor[1:]]
```

Comment:
I think you don't need the `... if '{}' in arg else arg` part - if the arg doesn't contain `{}`, `arg.replace('{}', ...)` will return the unchanged string anyways.